README.md

```diff
@@ -49,6 +49,12 @@ app.use(jwt({ secret: 'shhhhhhared-secret'}).unless({path: ['/token']}));
 
 This is especially useful when applying to multiple routes. In the example above, `path` can be a string, a regexp, or an array of any of those.
 
+If you are using a base64 encoded secret, simply allocate a new buffer and specify 'base64' encoding (before using the secret) like the following.
+
+```javascript
+var secret = new Buffer('shhhhhhared-secret', 'base64');
```

Comment:
@jfromaniello: is simply using a `Buffer` class like this enough for Auth0 client secrets or should we also perform the URL-decoding? It seems to work with just Buffer for me but maybe I'm missing something.